clients/java/src/main/java/io/camunda/client/CamundaClientBuilder.java

```diff
@@ -162,6 +163,8 @@ CamundaClientBuilder jobWorkerExecutor(
 
   CamundaClientBuilder withInterceptors(ClientInterceptor... interceptor);
 
+  CamundaClientBuilder withChainHandlers(AsyncExecChainHandler... chainHandler);
```

Comment:
🔧 Javadoc would be super helpful here to understand what to use this for and how. We can maybe add some for the interceptors above as well.